src/main/resources/templates/roomAvailability/edit.html

```diff
@@ -237,15 +263,23 @@ <h5 class="modal-title">Invalid Input</h5>
 
       });
     } else {
+<<<<<<< HEAD
       document.getElementById("day").textContent = "Monday";
+=======
+      document.getElementById("dropdownMenuButton").textContent = "Monday";
```

Comment:
Rather than making this a dropdown menu can we make this a modal?

Reply:
The dropdown has been changed to a modal.